src/main/kotlin/com/github/mitchellolsthoorn/testgenie/toolwindow/TestGenieToolWindow.form

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Perhaps I missed that because I am not in Delft with you. But this design will need to be confirmed with the team and the client right? 

I understand that we can change it later but did anyone see the design? Thanks.

Reply:
Well, we discussed that with our clients on Friday, and they approved our proposal.
I would say, we might change the structure/appearance of the actual UI components, but i think the actual elements like Tool Window, Settings, Selections remain the same (in the sense that they will be present)

Reply:
Also, again, this is just the template. Some of the team members will have to work on the actual form (with all the parameters) during the next sprint.